Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -209,7 +231,7 @@ public void DownloadAsync_Cancel()
                     try
                     {
                         var result = downloader.DownloadAsync(downloadUri, outputStream,
-                        handler.CancellationTokenSource.Token).Result;
+                                         handler.CancellationTokenSource.Token).Result;

```

Comment:
Thanks! (optional) I think that 4 spaces is enough. That's what we did in other places.. Except this one that you are fixing. Thanks for catching this one.